src/wyzeapy/utils.py

```diff
@@ -72,36 +72,45 @@ def create_password(password: str) -> str:
     return hashlib.md5(hex2.encode()).hexdigest()
 
 
-def check_for_errors_standard(response_json: Dict[str, Any]) -> None:
+def check_for_errors_standard(service, response_json: Dict[str, Any]) -> None:
     if response_json['code'] != ResponseCodes.SUCCESS.value:
         if response_json['code'] == ResponseCodes.PARAMETER_ERROR.value:
             raise ParameterError(response_json)
         elif response_json['code'] == ResponseCodes.ACCESS_TOKEN_ERROR.value:
-            raise AccessTokenError
+            service._auth_lib.token.expired = True
+            raise AccessTokenError("Access Token expired, attempting to refresh")
         elif response_json['code'] == ResponseCodes.DEVICE_OFFLINE.value:
             return
         else:
             raise UnknownApiError(response_json)
 
 
-def check_for_errors_lock(response_json: Dict[str, Any]) -> None:
+def check_for_errors_lock(service, response_json: Dict[str, Any]) -> None:
     if response_json['ErrNo'] != 0:
         if response_json.get('code') == ResponseCodes.PARAMETER_ERROR.value:
             raise ParameterError
         elif response_json.get('code') == ResponseCodes.ACCESS_TOKEN_ERROR.value:
```

Comment:
I would guess these are sending back an int based on ErrNo being an int, but I dont have a lock so I'm not sure. I don't want to change it if its working.

Reply:
Yeah, I probably wouldn't change it. I would just add your logic for refreshing the token prior to raising the exception. That was what I'd originally meant. Just that there were other locations that were raising the AccessTokenError exception, than just the one spot you'd originally modified ;)

Reply:
Got it. I think that I got all the check for error functions and the wrappers that use them all good to go now.